modules/core/src/test/java/org/apache/ignite/spi/discovery/tcp/TcpDiscoveryCoordinatorFailureTest.java

```diff
@@ -375,7 +375,7 @@ private boolean isDrop(TcpDiscoveryAbstractMessage msg) {
                 }
             }
 
-            if (drop)
+            if (drop && msg != null)
```

Comment:
@aleksey-kurinov , same here. Try removing `FailingDiscoverySpi.writeToSocket(Socket sock, TcpDiscoveryAbstractMessage msg, byte[] data, long timeout )`. Onve we do it, we'd not need this change. Am I right?